js/src/hooks/usePolicyCheck.js

```diff
@@ -11,12 +11,13 @@ import useAppSelectDispatch from './useAppSelectDispatch';
  *
  * ```json
  * {
- *	'policy_check' =>	[
- *				'allowed_countries'    	=> true,
- *				'store_ssl'         	=> true,
- *				'payment_gateways'  	=> true,
- *				'refund_returns' 	=> true,
- *				]
+ *		"allowed_countries": true,
+ *		"robots_restriction": false,
+ *		"page_not_found_error": false,
+ *		"page_restricts": false,
+ *   		"store_ssl": true,
+ * 		"payment_gateways": true,
+ *		"refund_returns": true
```

Comment:
Some indentations are still mixed with spaces and tabs. And I believe the key `page_restricts` should be `page_redirects`.

```suggestion
 *   "allowed_countries": true,
 *   "robots_restriction": false,
 *   "page_not_found_error": false,
 *   "page_redirects": false,
 *   "store_ssl": true,
 *   "payment_gateways": true,
 *   "refund_returns": true
```